frontend/public/src/components/CartItemCard.js

```diff
@@ -21,18 +27,19 @@ export class CartItemCard extends React.Component<Props> {
 
     const title =
       course !== undefined ? (
-        <a href="#" target="_blank" rel="noopener noreferrer">
-          {course.title}
-        </a>
+        this.courseAboutLink(course.title, course)
       ) : (
         <a href="#" target="_blank" rel="noopener noreferrer">
           {product.description}
         </a>
       )
 
+    const courseDetail =
+      this.courseAboutLink("Course details", course)
+
     const readableId =
       course !== undefined
-        ? purchasableObject.readable_id
+        ? purchasableObject.readable_id.split('+')[1]
```

Comment:
I think here we can use `course.readable_id` instead

Reply:
@annagav, from my understanding of the issue ticket the intention was to only display the course number and not the full id, which I assume the `readable_id` was showing the full id since that's what was in place.  Maybe I overlooked something somewhere.

Reply:
@annagav where does the formatting f `readable_id` take place?  When I look at RC, the course readable_id's that I see are all in the format of `XXXX:XXX+XXXX`.

Reply:
it's just a field on the model of `Course` and `Program`

Reply:
@annagav I'm confused though.  Does the `readable_id` provide any additional formatting so it only shows the course number like shown in https://github.com/mitodl/mitxonline/issues/871?

Reply:
If I understand it correctly, we want to display readable_id from the course, which is the format like  [course-v1:MITx+14.740x](https://rc.mitxonline.mit.edu/admin/courses/course/6/change/).

The purchasableObject.readable_id here refers to the courserun id (course-v1:MITx+14.740x+2T2022) https://rc.mitxonline.mit.edu/admin/courses/courserun/7/change/ if product is courserun

purchasableObject.readable_id.split('+')[1] will output '14.740x' if courserun id is like course-v1:MITx+14.740x+2T2022, which doesn't seem to be correct? I could be wrong though

Reply:
@rachellougee that could be, I guess I was looking at the screenshots included in issue #871 where currently `course-v1:Arbisoft+ARB_TEST_CKT_001+1` is shown, and we want to only show `ARB_TEST_CKT_001`.  But looking into RC I don't see anywhere that only `ARB_TEST_CKT_001` is stored or provided without some sort of pre-processing.

Reply:
@collinpreston Yeah, I noticed that. I think the test data on RC is causing confusion. The course corresponds to course-v1:Arbisoft+ARB_TEST_CKT_001+1 is https://rc.mitxonline.mit.edu/admin/courses/course/14/change/, which isn't ARB_TEST_CKT_001.  Not sure what the course number means, Is it course id? or does it has to be from the second part of courserun id?

Reply:
Just for the record, course_number is extracted from courserun.courseware_id  https://github.com/mitodl/ol-django/blob/main/src/mitol/openedx/utils/courses.py. so I was wrong

Reply:
All good.  Thanks for thinking through it with me @rachellougee